[ao_migration] torch/nn/quantizable: torch.quantization -> torch.ao.quantization

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #65903
* #65902
* __->__ #65901
* #65900

This changes the imports in the `caffe2/torch/nn/quantizable` to include the new import locations.

```
codemod -d torch/nn/quantizable --extensions py 'torch.quantization' 'torch.ao.quantization'
```

Differential Revision: [D31301194](https://our.internmc.facebook.com/intern/diff/D31301194/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31301194/)!

Commit:
[ao_migration] torch/nn/quantizable: torch.quantization -> torch.ao.quantization

This changes the imports in the `caffe2/torch/nn/quantizable` to include the new import locations.

```
codemod -d torch/nn/quantizable --extensions py 'torch.quantization' 'torch.ao.quantization'
```

Differential Revision: [D31301194](https://our.internmc.facebook.com/intern/diff/D31301194/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31301194/)!

[ghstack-poisoned]

albanD reviewed (commented):
CI doesn't seem happy with this one?

z-a-f commented:
> CI doesn't seem happy with this one?

Oh yea -- need to rebase. There was a fix

Commit:
Update on "[ao_migration] torch/nn/quantizable: torch.quantization -> torch.ao.quantization"


This changes the imports in the `caffe2/torch/nn/quantizable` to include the new import locations.

```
codemod -d torch/nn/quantizable --extensions py 'torch.quantization' 'torch.ao.quantization'
```

Differential Revision: [D31301194](https://our.internmc.facebook.com/intern/diff/D31301194/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31301194/)!

[ghstack-poisoned]

Commit:
Update on "[ao_migration] torch/nn/quantizable: torch.quantization -> torch.ao.quantization"


This changes the imports in the `caffe2/torch/nn/quantizable` to include the new import locations.

```
codemod -d torch/nn/quantizable --extensions py 'torch.quantization' 'torch.ao.quantization'
```

Differential Revision: [D31301194](https://our.internmc.facebook.com/intern/diff/D31301194/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31301194/)!

[ghstack-poisoned]

Commit:
Update on "[ao_migration] torch/nn/quantizable: torch.quantization -> torch.ao.quantization"


This changes the imports in the `caffe2/torch/nn/quantizable` to include the new import locations.

```
codemod -d torch/nn/quantizable --extensions py 'torch.quantization' 'torch.ao.quantization'
```

Differential Revision: [D31301194](https://our.internmc.facebook.com/intern/diff/D31301194/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31301194/)!

[ghstack-poisoned]

albanD reviewed (approved):
Looks good now!

Commit:
Update on "[ao_migration] torch/nn/quantizable: torch.quantization -> torch.ao.quantization"


This changes the imports in the `caffe2/torch/nn/quantizable` to include the new import locations.

```
codemod -d torch/nn/quantizable --extensions py 'torch.quantization' 'torch.ao.quantization'
```

Differential Revision: [D31301194](https://our.internmc.facebook.com/intern/diff/D31301194/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31301194/)!

[ghstack-poisoned]
